### PR TITLE
Hypermodules.Hyperamp version 0.6.3.241

### PR DESCRIPTION
**Pull request by oxygen-dioxide**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/21929)

### CODE REVIEW

**Comment by Trenly** [comment]
image

File: manifests/h/Hypermodules/Hyperamp/0.6.3.241/Hypermodules.Hyperamp.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.3.0.3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: Hypermodules.Hyperamp
+PackageVersion: 0.6.3.241
```
Comment (OfficialEsco) [inline manifests/h/Hypermodules/Hyperamp/0.6.3.241/Hypermodules.Hyperamp.installer.yaml:5]:
```suggestion
PackageVersion: 0.6.3
```

File: manifests/h/Hypermodules/Hyperamp/0.6.3.241/Hypermodules.Hyperamp.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.3.0.3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: Hypermodules.Hyperamp
+PackageVersion: 0.6.3.241
```
Comment (OfficialEsco) [inline manifests/h/Hypermodules/Hyperamp/0.6.3.241/Hypermodules.Hyperamp.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 0.6.3
```

File: manifests/h/Hypermodules/Hyperamp/0.6.3.241/Hypermodules.Hyperamp.yaml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 0.3.0.3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+
+PackageIdentifier: Hypermodules.Hyperamp
+PackageVersion: 0.6.3.241
```
Comment (OfficialEsco) [inline manifests/h/Hypermodules/Hyperamp/0.6.3.241/Hypermodules.Hyperamp.yaml:5]:
```suggestion
PackageVersion: 0.6.3
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]
SmartScreen don't seem to block validation anymore
image

**Review by OfficialEsco — APPROVED** [review]

**Pull request merged**